src/uwtools/api/cdeps.py

```diff
@@ -11,4 +11,4 @@
 _driver = CDEPS
 execute = _make_execute(_driver, with_cycle=True)
 tasks = partial(tasks, _driver)
-__all__ = [_driver.__name__, "execute", "graph", "tasks"]
+__all__ = ["CDEPS", "execute", "graph", "tasks"]
```

Comment:
Here and in many files that follow, use explicit strings in `__all__` so that static-analysis tools can see what this module is exporting.